### ISSUE TITLE
WebSocket is not defined

### ISSUE THREAD

**Issue by DolevAlgam** [Assembless/gather-town-assembless-api#2]
Hey,
I tried using this tool to report Gather participants to Slack.
After a lot of debugging I'm stuck with this error log.
Have you seen this before?

`⚡️ Gather is connected!`
`Failed to create websocket object with gameserver wss://game-aaai-025.gather.town:443 : {"message":"WebSocket is not defined","stack":"ReferenceError: WebSocket is not defined\n    at Engine.<anonymous> (/usr/src/app/node_modules/@gathertown/gather-game-client/dist/src/Engine.js:166:43)\n    at step (/usr/src/app/node_modules/@gath`
`t: 861 : destroying connection`
`t: 1861 : destroying connection`

Would appreciate help.
Thank You.

**Comment by DolevAlgam** [comment]
@DRFR0ST  🙏

**Comment by DolevAlgam** [comment]
It might be because I'm not using my Member ID, I didn't figure out how to find it.

**Comment by DolevAlgam** [comment]
With the help of @DRFR0ST this issue has been resolved.
As the log error suggest, there is a missing requirement in the Gather Town module.
the Engine.js file attempts to use the WebSocket command without requiring it.

This is already taken care of in this Repo [here](https://github.com/Assembless/gather-town-x-slack/blob/main/patches/%40gathertown%2Bgather-game-client%2B36.0.3.patch#L7).

For this to take effect I needed to perform the following command:
`yarn postinstall`
It immediately solved the problem.

Thanks @DRFR0ST .
